README.md

```diff
@@ -32,5 +32,40 @@ This is a blog site designed with microservices architecture using `Spring Cloud
 
 - **Config Server**: A Spring Boot application which provides configs (yml files) to all other services. Should be run first.
 - **Service Discovery**: A Spring Boot application with embedded `Service Discovery Server` (Eureka server). Each other services except `Config Server` are registered in it and can access each other by names instead of host-port combination using `Api Gateway Router` service. Should be run second.
-- **Frontend**: Spring Boot application with Angular parts. Contains embedded `Api Gateway Router` (Zuul), `Client Side Load Balancer` (Ribbon) and `Service Discovery Client` (Eureka client) which help to redirect requests to Api service instances registered in `Service Discovery Server`.
-- **Api**: Spring Boot application with backend information for the Frontend. Includes `Spring Security` parts to provide Json Web Tokens and validate them.
+- **Frontend**: A Spring Boot application with Angular parts. Contains embedded `Api Gateway Router` (Zuul), `Client Side Load Balancer` (Ribbon) and `Service Discovery Client` (Eureka client) which help to redirect requests to Api service instances registered in `Service Discovery Server`.
+- **API**: A Spring Boot application with backend information for the Frontend. Includes `Spring Security` parts to provide Json Web Tokens and validate them.
+
+## How to run?
+
+### Local environment
+
+1. Set environment variables
+
+       SPRING_PROFILES_ACTIVE=local
+       SPRING_CLOUD_CONFIG_SERVER_GIT_URI=<path-to-config-repo>
```

Comment:
This one is optional, right? With the 'local' Spring profile it has the default directory configured (which should be moved out of the config server app to the root level).

Reply:
You can set SPRING_PROFILES_ACTIVE=local,native.
But only Config Server requires the second 'native' profile.

Another option is to run apps as Java applications and specifying Spring profile as a JVM parameter:
java -Dspring.profiles.active=local,native -jar config-server/build/libs/config-server-1.0.jar